### PR TITLE
DOCSP-31487: removed options for command() v6

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-31487>
Staging
- [what's new](https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-31487-command-options-changes/whats-new/#what-s-new-in-6.0)
- [upgrade guide](https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-31487-command-options-changes/upgrade/#version-6.x-breaking-changes)
- [run a command fundamentals](https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-31487-command-options-changes/fundamentals/run-command/#command-options)
- [run a command usage ex](https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-31487-command-options-changes/usage-examples/command/)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: source/fundamentals/run-command.txt
Comment (ccho-mongodb) [inline source/fundamentals/run-command.txt:66]: Suggestion:

1) I think it would be more clear to specify the command methods rather than expecting the reader to associate it with content from the middle of the previous section.

2) I think "set option ... by passing an options object" is redundant.

3) I think the association between "options" object and the two types of options objects could be more clear.


```suggestion
You can specify optional command behavior for either the ``command()`` or the ``runCursorCommand()`` method.

The ``command()`` method accepts a RunCommandOptions object. To learn more about the RunCommandOptions type, see the API documentation.

The ``runCursor() method`` accepts a ``runCursorCommand()`` object. To learn more about the RunCursorCommandOptions type, see the API documentation.
```

File: source/fundamentals/run-command.txt
Comment (ccho-mongodb) [inline source/fundamentals/run-command.txt:78]: Suggestion:

1) I think the reader would benefit from a clear differentiation between the command options described in the first paragraph and "command-specific options".

I also think that it might not be clear which options can be specified in the command document, and whether it depends on the command.

To contrast this with the "only the following options" from the prior paragraph:
"You can specify additional commands on certain commands..." 


2) I think the mention of write or read concerns as options that you can specify in the command document, followed by an example of maxTimeMS is confusing. To resolve this, provide the reader with a reference that they can check to determine which options they can provide which commands.

File: source/fundamentals/run-command.txt
Comment (ccho-mongodb) [inline source/fundamentals/run-command.txt:69]: Question: Does this apply to the runCursorCommand() method or only command() method? In either case, I think it could be good to call out the limitations for both methods.

File: source/fundamentals/run-command.txt
Comment (ccho-mongodb) [inline source/fundamentals/run-command.txt:94]: Suggestion:

I think it would be good to avoid humanizing inanimate objects when possible (Use effective verbs from the style guide).

"The ... methods ignore" or "The ... methods throw an error if..."

Also, in the next sentence, I think "earlier" is more correct than "elsewhere".

**Review by ccho-mongodb — CHANGES_REQUESTED** [review]
Looks pretty good! I identified couple issues and suggestions. Would like to re-review once addressed.

File: source/upgrade.txt
Comment (rustagir) [reply source/upgrade.txt:74]: I assume that since the options are being **removed**, whereas they were previously just deprecated, the driver will throw an error.

Comment (ccho-mongodb) [reply source/upgrade.txt:74]: I think "take" is an issue related to [Don't Humanize Inanimate Objects](https://www.mongodb.com/docs/meta/style-guide/writing/use-effective-verbs/#don-t-humanize-inanimate-objects) (the style guide examples aren't the clearest) because it implies performing an action that is usually associated to an intention.

I think "accept" would be better. I think "command-specific options" also could be substituted for a more definitive description.

Suggestion:

"The Db.command() method only accepts [describe the options it accepts] options. To learn more about these options, see ..."

File: source/fundamentals/run-command.txt
Comment (rustagir) [reply source/fundamentals/run-command.txt:74]: the ticket only says that it is for the `command()` method and from the feature PR, it seems  that `runCursorCommand()` is unchanged.

File: source/upgrade.txt
Comment (ccho-mongodb) [inline source/upgrade.txt:73]: Issue:

Perhaps my fault for providing this in the suggestion, but I think "only" is a misplaced modifier in this case.
From the [style guide on "only"](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#o):
> Place only preceding the word or group that it modifies. If only modifies a final word or phrase, it may precede or follow that word or phrase.

Suggestion:

I think In this case, "only" modifies options rather than "accepts".

```suggestion
- The ``Db.command()`` method accepts only options that are not related
```

File: source/fundamentals/run-command.txt
Comment (ccho-mongodb) [inline source/fundamentals/run-command.txt:85]: Issues:

1) I think this could be confusing to a user since the page this links to shows the overview of runCommand() and adminCommand() rather than command() and "its associated options". I also wasn't able to locate the aforementioned write or read concern options.

2) The text of the link should be descriptive. See [Link and Cross-Reference Examples](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#link-and-cross-reference-examples).

Suggestion:

From looking at docs/manual/reference/command, it looks like the options are listed only on the target pages of the links of each command. It could be helpful to inform the reader that is how it is structured to reduce the chance of confusion. Also, I think it would be good to continue using the format "\<Page title\>" in the Server Manual.

E.g.
"Learn more about the available options for each command by following the link associated with the command in the :manual:\`User Commands </reference/command/#user-commands>\` section of the Server Manual"

File: source/fundamentals/run-command.txt
Comment (ccho-mongodb) [inline source/fundamentals/run-command.txt:104]: Suggestion:

I think it would be easier to understand and more concise by using [Positive Statements](https://www.mongodb.com/docs/meta/style-guide/writing/use-positive-statements/):

```suggestion
   The ``command()`` and ``runCursorCommand()`` methods ignore the
   read preference setting you may have set on your ``Db`` object.
```

I don't think it's necessary to mention "earlier in your code" since "you may have set" refers to the recent past.

File: source/usage-examples/command.txt
Comment (ccho-mongodb) [inline source/usage-examples/command.txt:10]: Suggestion:

I think the second sentence attempts to fit too much information. I also think this could be more clear if broken down into communicating the following info in order:

1) The command() method allows you to run any MongoDB database operation. This is an instance method of the Db class.

2) To specify a command, create a command document that follows the command syntax (https://www.mongodb.com/docs/manual/reference/command/#command-syntax).

3) To run the command on the database, pass the command document to the command() instance method.

Let me know if you need any help crafting this description from this breakdown.

**Review by ccho-mongodb — CHANGES_REQUESTED** [review]
I think this is nearly there, but have a few suggestions and would like to review a couple changes.

File: source/fundamentals/run-command.txt
Comment (rustagir) [reply source/fundamentals/run-command.txt:85]: since the page describes commands outside of the User Commands category (such as `hello`), I will change the link and link text slightly

File: source/usage-examples/command.txt
Comment (rustagir) [reply source/usage-examples/command.txt:10]: Since this is a usage example, I dont think we should describe the command syntax. I think it is sufficient to mention that users need to create a command document and then link out to the manual

File: source/fundamentals/run-command.txt
Comment (ccho-mongodb) [inline source/fundamentals/run-command.txt:90]: Question:
Did you mean to include this? If so, it repeats same the in the previous paragraph (style guide issue and confusing user experience) and should be removed.

File: source/usage-examples/command.txt
Comment (ccho-mongodb) [reply source/usage-examples/command.txt]: I think we're in agreement. I meant was that this should describe the method syntax (which it currently does, but could be more clear), not the command syntax. The important part is to mention that it is a document that contains a command because that's the required parameter.

File: source/fundamentals/run-command.txt
Comment (ccho-mongodb) [reply source/fundamentals/run-command.txt]: Resolving and adding a comment to the revised version.

**Review by ccho-mongodb — CHANGES_REQUESTED** [review]
Looks good, only a couple items left to address.

**Review by ccho-mongodb — APPROVED** [review]
Sorry, I just noticed an issue about the example. Could you address that before merging? Otherwise, I provided a couple suggestions and think this shouldn't require another review, but happy to take another look if you want.

**Pull request merged**
